rllib/utils/tests/test_check_env.py

```diff
@@ -37,6 +38,7 @@ def test_obs_and_action_spaces_are_gym_spaces(self):
         with pytest.raises(
                 ValueError, match="Action space must be a gym.space"):
             check_env(env)
+        del env
```

Comment:
the old review comments are lost (I can't find it anywhere). so start a new thread.
I wonder if this is necessary. not saying one way or another, but we should be consistent.
if there is a real need for explicitly deleting 3rd party env, we should do this everywhere.

Reply:
whoops sorry about that -- I resolved it because I thought I had answered your question. 

My answer was this:
"no magic here. Just habit, for preventing memory leaks in tests, a habit I formed when developing [Metaworld](https://github.com/rlworkgroup/metaworld)"

Reply:
sure, do you want to keep this habit?
I kind of think consistency is very important for a codebase. if there's actual benefit, we should use this convention everywhere.
otherwise, maybe we shouldn't del it explicit here either.

Reply:
Yeah, doing this doesn't make sense to me either. I mean, even if there was a memory leak in a test case like this, it wouldn't matter, right?

Reply:
Ok I'll go ahead and remove it. Thanks for the feedback.

Reply:
Let's leave it for now and merge this! :)
We can discuss offline:
* Do or not do `del` in test cases.
* Whether we should consistently use `unittest` or `pytest`. I heavily prefer `unittest` as it's much easier to debug with in PyCharm (pytest does not produce output in the interactive shell in the debugger, etc..). But happy to learn otherwise.

Reply:
if you use "pytest -s ...", it will stream the output while running the test.